Homework 2/Task 2.3/Lecture.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Homework_2.Task_2._3
+{
```

Comment:
Think to implement IClonable interface for Lecture, Practical lecture and Training